consumers/email-consumer/main.py

```
@@ -0,0 +1,60 @@
+        if msg is None:
+            continue
+        if msg.error():
+            print("Consumer error: {}".format(msg.error()))
+            continue
+        print(f'Message: {msg.value().decode("utf-8")} partition: {msg.partition()}')
+        try:
+            msg_data = json.loads(msg.value())
+            send_verification_email(**msg_data)
+        except json.JSONDecodeError:
+            print("Failed to decode message")
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)